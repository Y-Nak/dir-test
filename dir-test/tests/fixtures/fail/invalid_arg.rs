use dir_test::dir_test;

#[dir_test(
    dir: "/foo/bar",
    loader: std::fs::read_to_string
    dir: "Dup"
)]
fn foo(fixture: Fixture<&str>) {}

#[dir_test(
    dir: "/foo/bar",
    foo: "fooBar",
)]
fn foo(fixture: Fixture<&str>) {}

fn main() {}
