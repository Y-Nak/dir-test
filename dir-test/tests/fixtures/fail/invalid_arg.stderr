error: duplicated arg `dir`
 --> tests/fixtures/fail/invalid_arg.rs:6:5
  |
6 |     dir: "Dup"
  |     ^^^

error: unknown arg `foo`
  --> tests/fixtures/fail/invalid_arg.rs:12:5
   |
12 |     foo: "fooBar",
   |     ^^^
